unfetter-discover/unfetter-ui
# Issue 644/assess results view

supports unfetter-discover/unfetter#644
wip view assessment results page.  Some routing works, some styles applied.  Add assessment does not work, currently selected item is not styled.

#To Test
* Navigate to Assessment 2.0
* Create and Assessment
* Click the `Full Results` tab
* see content, and verify no error in console
* `npm run lint && npm run build:dev && npm run test` verify no errors
